src/Utils/request/api.tsx

```diff
@@ -573,6 +580,53 @@ const routes = {
       },
     },
   },
+
+  consent: {
+    create: {
+      method: "POST",
+      path: "/api/v1/patient/{patientId}/consent/",
+      TRes: Type<ConsentModel>(),
+      TBody: Type<CreateConsentRequest>(),
+    },
+    list: {
+      method: "GET",
+      path: "/api/v1/patient/{patientId}/consent/",
+      TRes: Type<ConsentListResponse>(),
+    },
+    retrieve: {
+      method: "GET",
+      path: "/api/v1/patient/{patientId}/consent/{id}/",
+      TRes: Type<ConsentModel>(),
+    },
+    update: {
+      method: "PUT",
+      path: "/api/v1/patient/{patientId}/consent/{id}/",
+      TRes: Type<ConsentModel>(),
+      TBody: Type<UpdateConsentRequest>(),
+    },
+    delete: {
+      method: "DELETE",
+      path: "/api/v1/patient/{patientId}/consent/{id}/",
+    },
+    addVerification: {
+      method: "POST",
+      path: "/api/v1/patient/{patientId}/consent/{id}/add_verification/",
+      TRes: Type<ConsentModel>(),
+      TBody: Type<{ verification_type: VerificationType }>(),
+    },
+    removeVerification: {
+      method: "POST",
+      path: "/api/v1/patient/{patientId}/consent/{id}/remove_verification/",
+      TRes: Type<ConsentModel>(),
+      TBody: Type<{ verification_id: string }>(),
+    },
+    upsert: {
+      method: "POST",
+      path: "/api/v1/patient/{patientId}/consent/upsert/",
+      TRes: Type<ConsentModel>(),
+      TBody: Type<CreateConsentRequest>(),
+    },
+  },
```

Comment:
new way of doing things is to have a dedicated file: `consentApi.ts` and keeping it there instead;